src/client/content_cao.cpp

```diff
@@ -1617,12 +1617,10 @@ void GenericCAO::processMessage(const std::string &data)
 						v2f(m_prop.visual_size.X, m_prop.visual_size.Y) * BS);
 				m_env->addSimpleObject(simple);
 			} else if (m_reset_textures_timer < 0) {
```

Comment:
same for the other
```suggestion
			} else if (m_reset_textures_timer < 0 && !m_prop.damage_texture_modifier.empty()) {
```

Reply:
Is this to account for mods explicitly overriding the default texture modifier with `""`?

Reply:
Yes, it works fine without this but there is no need to set the texture timers and whatnot when the mod wants the damage texture disabled.

Reply:
Ahh, understood. Thanks.